scio-core/src/main/scala/com/spotify/scio/values/SCollection.scala

```diff
@@ -769,18 +769,21 @@ sealed trait SCollection[T] extends PCollectionWrapper[T] {
     out: () => PrintStream = () => Console.out,
     prefix: String = "",
     enabled: Boolean = true
-  ): SCollection[T] =
+  )(implicit coder: Coder[T]): SCollection[T] =
```

Comment:
Since the `SCollection` keeps the same type, the implicit param is not strictly necessary. You coud get it from `internal.getCoder`. It would avoid a change in type signature.

Reply:
ah right! yes no need for this neither for the `tap` nice catch!